snuba/migrations/system/0001_migrations.py

```diff
@@ -0,0 +1,29 @@
+from typing import Sequence
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migrations
+from snuba.migrations import operations
+
+
+class Migration(migrations.Migration):
+    is_dangerous = False
+    dependencies = None
+
+    def forwards_local(self) -> Sequence[operations.Operation]:
+        return [
+            operations.RunSql(
+                storage_set=StorageSetKey.MIGRATIONS,
```

Comment:
It's only for getting the right cluster. I wanted to make a migration as flexible as possible and allow for operations that are not in the same storage set to be defined as part of the same migration.

Reply:
Not sure it adds a lot of flexibility since it does not seem very apparent to me why a migration would span multiple storage sets for a legitimate use case. 
Otherwise, making the interface of a component stricter tends to restrict its responsibility, what the implementation class should care about, thus harder to make mistakes. 
Though thinking over this, adding it to the operations seems the only viable approach:
- if it is a attribute of the migration the operation would need to refer to the migration class
- a migration may not care about the database, there is no reason for it to be associated to a storage set 
- passing it from migration to operation in the execution method would equally not make sense since the operation may not care about the DB.

So I think it should stay where it is.